netlify.toml

```diff
@@ -1,99 +1,111 @@
-########################################
+########################################################################################################################
 # General redirects
-########################################
+########################################################################################################################
 # Ensure old preview urls are redirected to the primary domain
 # https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects
 [[redirects]]
-  from = "https://preview.documentation.bonitasoft.com/*"
-  to = "https://documentation.bonitasoft.com/:splat"
-  force = true
+from = "https://preview.documentation.bonitasoft.com/*"
+to = "https://documentation.bonitasoft.com/:splat"
+force = true
 
 
-########################################
+########################################################################################################################
 # Bonita redirects
-########################################
+########################################################################################################################
+# latest and next: ONLY CHANGE WHEN THE BONITA GA IS OUT.
```

Comment:
Currently, it has been updated to early in https://github.com/bonitasoft/bonita-documentation-site/pull/392/files
- latest: 2022.2 beta
- next: 2023.1 alpha

latest should be the latest public release (GA or maintenance), not beta
next should be the upcoming release

This is mentioned in the Bonitasoft internal release process but not always followed. Adding a comment provides an additional guard.